release-scripts/tests/test_helm_artifacts.py

```diff
@@ -0,0 +1,31 @@
+import os
+import yaml
+import requests
+
```

Comment:
I don't see this script being called from anywhere. Is it documented to run it somewhere?

Reply:
The post-release tests are run by 
```
make release-test
```
which is described in the "Verifying the release" section of `RELEASING.md`.